spring-integration-core/src/main/java/org/springframework/integration/handler/AbstractReplyProducingMessageHandler.java

```diff
@@ -147,7 +147,19 @@ protected Object doInvokeAdvisedRequestHandler(Message<?> message) {
 
 	}
 
-	private class AdvisedRequestHandler implements RequestHandler {
+	/**
+	 * Used to wrap the
+	 * {@link AbstractReplyProducingMessageHandler#handleRequestMessage(Message)}
+	 * method. Also allows access to the underlying
+	 * {@link AbstractReplyProducingMessageHandler} to obtain properties.
+	 *
+	 * @author Gary Russell
+	 * @since 4.3.2
```

Comment:
`@since 4.2.10` I guess. 
Although I guess we can live in this inner class without version.